package.json

```diff
@@ -54,5 +54,8 @@
   "packageManager": "npm@8.15.0",
   "engines": {
     "node": "=18"
+  },
+  "dependencies": {
+    "turbo-darwin-64": "^1.4.3"
```

Comment:
don't add this to deps, the install of this might fail on linux or windows. just install it globally

Reply:
there should be a better solution to this problem, but for now, it works

Reply:
removed dependency